src/prisma-generator.ts

```diff
@@ -42,9 +43,33 @@ export async function generate(options: GeneratorOptions) {
   await enumsObj.printEnumSchemas();
 
   const inputObjectTypes = prismaClientDmmf.schema.inputObjectTypes.prisma;
+  const outputObjectTypes = prismaClientDmmf.schema.outputObjectTypes.prisma;
+
+  outputObjectTypes.forEach((outputObjectType) => {
+    if (isAggregateOutputType(outputObjectType.name)) {
+      const name = outputObjectType.name.replace(/(?:OutputType|Output)$/, '');
+      inputObjectTypes.push({
+        constraints: { maxNumFields: null, minNumFields: null },
+        name: `${name}Input`,
+        fields: outputObjectType.fields.map((x) => ({
```

Comment:
Please change x to field, since you're mapping over fields

Reply:
I have changed it. Thank.